engine/src/main/scala/cromwell/engine/workflow/lifecycle/ShadowMaterializeWorkflowDescriptorActor.scala

```diff
@@ -216,7 +216,7 @@ class ShadowMaterializeWorkflowDescriptorActor() extends LoggingFSM[ShadowMateri
                                    coercedInputs: WorkflowCoercedInputs,
                                    engineFunctions: WorkflowEngineFunctions): ErrorOr[WorkflowCoercedInputs] = {
     // TODO: Need to create engine-only engine functions!
```

Comment:
Yes, I mean currently they exist but they're kinda hidden. I'm creating "real" ones in the scatter support branch since we need them also to evaluate scatter collections.